adi/cn0511.py

```diff
@@ -120,3 +120,12 @@ def calibrated_output(self, value):
             )
         else:
             print("Warning: Missing libad9166, calibration failed.")
+
+    @property
+    def board_calibrated(self):
+        """ board_calibrated: 1 if board was calibrated in production, 0 if board was not calibrated in production"""
+        if libad9166:
+            return libad9166.device_is_calibrated(self.__calibration_data)
+        else:
+            print("Warning: Missing libad9166, calibration failed.")
+            return None
```

Comment:
Just a small nit pick, Since the above has a return and Python defaults to return None you only need 1 line of the 3. So:

```python
        if libad9166:
            return libad9166.device_is_calibrated(self.__calibration_data)
        print("Warning: Missing libad9166, calibration failed.")
```